docs/components/concepts/document-handling.md

```diff
@@ -0,0 +1,56 @@
+---
+id: document-handling
+title: "Document handling"
+description: "Learn more about integration, secure management, and efficient storage and retrieval of documents across development and production environments."
+keywords: ["document handling"]
+---
+
+Offering a more comprehensive approach to document handling, Camunda now provides efficient storage and retrieval of documents across both development and production environments.
+
+Whether you are working across Connectors, Modeler, or Tasklist, document handling may be beneficial for users who want the following:
+
+- [AWS S3](https://aws.amazon.com/s3/) storage and bucket creation per cluster to securely store and retrieve documents in an external, scalable storage solution, and to ensure storage is properly isolated and managed for each environment.
+- Support for the local file system as a storage option for development and testing purposes to set up and manage documents in local environments.
+- Preview, track, and manage documents associated with process instances.
```

Comment:
I wonder if this should be "The ability to support, track...", or similar, to keep the grammar of each bullet consistent.